src/helpers.rs

```diff
@@ -72,6 +72,34 @@ pub(crate) fn parse_field_attributes(attributes: &[syn::Attribute]) -> HashMap<S
     attrs
 }
 
+// Returns true if union has the repr(C) attribute set.
+pub(crate) fn check_union_layout(attributes: &[syn::Attribute]) -> bool {
```

Comment:
Looks like we can write some testing for this method.

Reply:
Yup.

Reply:
Added unit test. Initially I wanted to do this in the versionize crate as an integration test.